content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -460,23 +460,23 @@ The Automation beta includes the option to create a one-time email campaign, sim
 1. Find the list you want to export and hover over the action menu.
 1. Click **Export**.
    The contact list begins downloading, and you will receive a notification email to the email address specified in Marketing Campaigns Notifications. Go to your inbox to download the file.
-1. Once you download the file, navigate back to Automation.
+1. Once you download the file, navigate back to Automations.
 1. Select **Contacts**.
 1. To add the contacts, click **Upload Contacts**.
-1. Follow the instructions outlined in [Upload contacts]({{root_url}}#upload-contacts) to complete the process.
+1. Follow the instructions outlined in [Upload contacts]({{root_url}}#upload-a-csv) to complete the process.
 
 *To migrate a segment from Marketing Campaigns:*
 
-During beta, segmentation is not enabled in Automation. You can build a segment in Marketing Campaigns, export it, and import it as a list to Marketing Campaigns.
+During beta, segmentation is not enabled in Automation. Instead, you can build a segment in Automation, export it, and import it as a list to Marketing Campaigns.
```

Comment:
I think we need to rework this one. We start by saying it's unavailable in Automation, but then tell them to build one in automation. :)

Reply:
Haha yeah, woops! Thanks for catching that.